plugins/obs-outputs/librtmp/rtmp.c

```diff
@@ -242,8 +242,10 @@ int
 RTMPPacket_Alloc(RTMPPacket *p, uint32_t nSize)
 {
     char *ptr;
+#if __WORDSIZE == 32
```

Comment:
___WORDSIZE doesn't exist on microsoft's compilers.  I'd probably recommend including `<util/platform.h>` and then using `ARCH_BITS` instead, which defines the same value on all operating systems.